test/fixtures/docker-app/features/run_docker_services.feature

```diff
@@ -12,14 +12,17 @@ Feature: Running docker services
         Given I have built the service "test_1"
         When I start the service "test_1"
         Then the service "test_1" should be running
+        And the last run docker command exited successfully
```

Comment:
I don't think these features are being run currently.  I believe there would need to be an entry in `docker-compose.yml` and a separate build step in the pipeline to run them.

Reply:
I'll double check this, they _should_ be, but I can imagine they may not be